docs/getting-started-v2-beta.md

```diff
@@ -0,0 +1,109 @@
+---
+layout: page-with-side-nav
+title: Getting Started
+---
+# Getting Started
+
+De 'Bevraging Persoon' Web API is gespecificeerd in OpenAPI specifications (OAS).
+
+Wil je de API gebruiken? Dit kun je doen:
+
+1. Bekijk de [functionaliteit en specificaties](#functionaliteit-en-specificaties)
+2. [Implementeer de API Client](#implementeer-de-api-client)
+3. [Probeer en test de API](#probeer-en-test-de-api)
+
+## Functionaliteit en specificaties
+
+Je kunt een visuele representatie van de specificatie genereren met [Swagger UI]({{ site.baseurl }}/swagger-ui) of [Redoc]({{ site.baseurl }}/redoc).
```

Comment:
ik denk dat deze links niet meer werken. Hiervoor moeten we denk ik ook specifieke swagger-ui en Redoc bestandjes maken

Reply:
Klopt. Voor de API-LAB moeten deze ook verwijzen naar de specificatie van de v2.0.0beta-release. 
Daarnaast wordt polymorfisme alleen in Redoc gerepresenteerd. Misschien dat ook kort toelichten ?

Reply:
aangepast. Swagger UI link verwijderd

Reply:
aangepast